backend/organization/models/tags.py

```diff
@@ -0,0 +1,70 @@
+from django.db import models
+
+from organization.models import Organization
+
+
+class OrganizationTags(models.Model):
```

Comment:
Isn't this the same as `OrganizationTypes`? I think `OrganizationTypes` didn't exist in Evan's model, so I thought you put it instead of `OrganizationTags`

Reply:
Yeah I realized that after the code push. Removing `OrganizationTypes`.